Themes/README.md

```diff
@@ -1,5 +1,21 @@
L1yeV8B1EEPegz/uOmdCMo7iXIC5miGuDODqnYNsigrjWcJdMZIPYhp+/vxJ5i4qTxH3VkfMIHs8ZzsM85vK2TUEs+8PnoBJs1txBnNX2Npnr1QXxmY5dfR3Nr+E1KQhiAD3q/ZfqUssmCOIBkEiMg9iEb2otbziLW8R0IojtDHQulHNriJNx7thjxmuIy8vB4yjiurIeOfu42hvmL9XZi1o1iL34kaCKYrAIpEXE0GzHMUFcRpKboZwzkszjmi2Iy/O0Id9NENvHnY1aJ3Ps2iA+27HDHypyiucu9Vz54gBeSBDbv/+VfyUCgBl1GsT++uHUGmIJZH+/jK7fj3i0QWzC0S2XSESj5WK3GF6k+rfX35zkeosKYhHeb3QM900xtU8rjluMnaeJGWIt+7hmsPC3XWthriDWTCRJxMjQt/8qFXk2lHP/TJ5m48gb+XhMcCFqx+MHM0RScL4y/HP2gi8ZxPY5Km9r35GifM6qj0tG+dgq9y2j7XNWCL4mepRfv6Zje48nZ+5ja+HzokflvtoFcfA10BYSxPpvqvwAPO/fWQBwep0hHiJmiAFgOSoRfGb2X32YIQZwRgRxhCAGgOEr/1WHGAawAARxhCAGAAAYF4I4QhADAACMC0EcIYgBAADGhSCOEKUAAADjQhBHCGIAAIBxIYgjBDEAAMC4EMQRghgAAGBcCOJI30Fcfqoa760JAADQiU6DWD6CWT6K2X0sc/zRzfJn2eb2X5k8tnu6M4wZ4uLTl3ae2osAAABYml5niCV4/ei99eSZGeL09Vt14erd8nJXhrJkQj6FaXNyYi8BAABgWXoNYondulngOJi7QBADAACMS29B7JZHPDr+zm6pIogJYgAAgGXrPIglgHNriH1yPVlvLOuOu1QG8ctD9dHNfXVu91C9KraEjr9U52T//rd2Q+hoX+/T+6+l+rrp2OLpjtq4tKdIYgAAgOXqdcmERK+sE5b1wj73y3d1s8fLMpggFhLFGxvMFAMAACxRr0EsIXzx+iSYBXYx3PUv0zlDWjLBW68BAAAsX++/VOcvm+g7hgVriAEAAMal0yD21w+n1hDLL9D5+2V0vY6YIAYAABiXXmeIh4ggBgAAGBeCODKMIOaT6gAAALpCEEf6DuKD7Q3zzhK85RoAAEA3COLIUJZMAAAAoBsEcYQgBgAAGBeCOEIQAwAAjAtBHCGIAQAAxoUgjhDEAAAA40IQRwhiAACAcSGIIwQxAADAuHQaxPIRzPJRzO5jmeOPbo4/2lk+yrlrKx/ET/UX9ZJ8tMfQHKgdeX9lO/gUPgAAMBS9zhBL8OaiV0JZgvnWk2d2SzcI4mUoPnmvjOAXe2pTRzGfxAcAAIag1yCW2K2bBZZ9YwviPR2zEooH2/qLo786MvZe2J1CgtduD/bp/2562/1xYK/iju2cTPR+fT+Gvb1cd8fd1gtrue6mHv55tQ7apzv6+jvleQjziXzb/ha7LboeAADAsums6YebAZZlEimnr9+qi9cnZplFl4YQxBKbEp/CBGgUrWUE2zgOAlK2ZWaI2wSxHM8dX8LYXd9cV19252UuZ+4ndjLZDOLXXNYHiz+emiAGAAB90InTLX+dcLyGWMiM8CDWEL88VB/d3Ffndg/Vq2JL6PhLdU72739rN4SO9vU+vf9aqq9rjm2C2EWq8ALXzdL64sg9axD7+yXGgwD2z8teP4jXzOMqg9jMFNsQNssmiF8AANC/zoPYJ3F84epdMxucInGciuZlGkIQB4Hr8Zcr+GORQZwL1DMHsT7RYEZY4jiaIQYAAOiDTpr+NC2LkO2yPxfMyzCUNcQp/oxtVp9BnJNYQxwvowAAAOhLr0HcNANc9y4UyzLkIDaxq79iwS/ZxWyopq4jxy6D2h6rkyDW1zJvuVYGcHE5fpzFGuLN+scHAACwYDppuhO/z3Acw/76YRm9riHuSW0QCxey3oij1MRrar+NWLNd389BVzPERvg+xKnH6JZW8B7FAACgSzpp4Os7iEfNLK1ghhgAAHSLII4QxH1ws8fEMAAA6B5BHCGIAQAAxoUgjhDEAAAA40IQRwhiAACAcSGIAQAAMGoEMQAAAEaNIAYAAMCoEcQAAAAYNYI4wX1i2salPcVnpgEAAKy3ToP46PmpOr91J/vRzY5sk31yXblNXw62+RhhAACAddfrDPGVyWMzYreePDNBfPH6pNcgNh8lvH1gLwAAAGAd9RrEEr5xEEsAf/jJrnp0/B1BDAAAgKXrLYjdsggJX8dtk1A+ff2WIAYAAMDSdR7EEsC5NcSyz20bRBC/2FObGzuKJAYAAFhfvS6ZkAC+cPWuid84gAcRxMJE8QYzxQAAAGuq1yD2o9efOY5H6hfvOiFLJnjrNQAAgLXW+y/VxcsmnEHMELOGGAAAYO11GsTxLHAuhgVBDAAAgC70OkM8eAQxAADA2iOIa/BJdQAAAOuPIE44mWyqDXlnCd5yDQAAYO0RxAAAABg1ghgAAACjRhBHiqUSG+rXv/41g8FgMBgMBmMEgyCOEMQMBoPBYDAY4xoEccQFMQAAAMaB8osQxAAAAONC+UUIYgAAgHHptPy+/Z+n6n/7cEv9L//rH5rxX//wM/X//cf0o5vj/TL+r//7p3ZvNwhiAACAcem1/CR2/eCVIP7D//O/BZHcNYIYAABgXHotv3v/7X8QxAAAAOhVb+Un0StLJr76H//TbiGIAQAA0L3Oy08CuO0a4v/9//ix+tdfv7V7u0EQAwAAjEuv5SdxXBe9sqSi6ygmiAEAAMal1/KT0N38r3fMzHCKbJf9BDEAAACWpdfykxngeNmE49YY87ZrAAAAWKZOy89fP5xaQyyB7O+Xy10jiAEAAMaF8osQxAAAAONC+UUIYgAAgHGh/CIEMQAAwLhQfhEXxL/+9a8ZDAaDwWAwGCMYBHGEIGYwGAwGg8EY1yCIAQAAMGoEMQAAAEZMqf8f2+qi2O7co58AAAAASUVORK5CYII=" alt="" />
```

Comment:
Most of the other extensions I've seen do not embed the image data in the readme.  They link to a resource online or in the VSIX package.  Maintenance-wise, it seems like it would be easier if we just created a "resources" folder and stored the .pngs in there?